src/scifem/biomedical.py

```diff
@@ -80,19 +103,28 @@ def read_mri_data_to_function(
     mri_data_path: str | Path,
     mesh: dolfinx.mesh.Mesh,
     cells: npt.NDArray[np.int32] | None = None,
+    vox2ras_transform: npt.NDArray[np.floating] | None = None,
     degree: int = 0,
     dtype: np.inexact = dolfinx.default_scalar_type,
+    use_tkr: bool = False,
 ) -> dolfinx.fem.Function:
     """Read in MRI data over a set of cells in the mesh and attach it to an appropriate function.
 
     Args:
         mri_data_path: Path to MRI data
         mesh: The mesh to attach the MRI data to.
         cells: Subset of cells to evaluate the MRI data at. If None, all cells are used.
+        vox2ras_transform: Optional transformation matrix to convert from voxel to ras coordinates.
```

Comment:
Should we sanity check the size of this transform (4x4), or let nibabel handle it?